### PR TITLE
Fix/remove ports from test config

### PR DESCRIPTION
**Pull request by gunther-bachmann**

### CODE REVIEW

File: org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend [hunk 1]
```diff
@@ -30,41 +27,26 @@ import org.testeditor.tsl.dsl.web.TslWebSetup
...
-	 * https://github.com/HubSpot/dropwizard-guice/issues/95
-	 * https://github.com/HubSpot/dropwizard-guice/issues/88
-	 * https://github.com/Squarespace/jersey2-guice/pull/39
-	 */
-	@BeforeClass
-	static def void ensureServiceLocatorPopulated() {
-		JerseyGuiceUtils.reset
-	}
+	@Rule
+	public val wiredMockRule = new WireMockRule(wireMockConfig.port(0))
```
Comment (jjelschen) [inline org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend]: port(0) indicates a "random, free port"? Maybe add a (very) brief comment(?)

File: org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend [hunk 2]
```diff
@@ -30,41 +27,26 @@ import org.testeditor.tsl.dsl.web.TslWebSetup
...
-
-	@Rule
-	public val dropwizardServer = new DropwizardClientRule(dummyResource);
-
 	@Before
 	def void registerCustomSerializers() {
-		val customSerializerModule = new SimpleModule
-		customSerializerModule.addSerializer(IEObjectDescription, new EObjectDescriptionSerializer())
-		dropwizardServer.objectMapper.registerModule(customSerializerModule)
+		stripLingeringMetrics(dropwizardClient.getEnvironment());
```
Comment (jjelschen) [inline org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend:49]: This workaround is still required?

File: org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend [hunk 3]
```diff
@@ -87,25 +78,15 @@ class IndexServiceClientIntegrationTest {
 
 		// then 
 		assertThat(actual.allElements).satisfies [
-// will not compile with Xtext < 2.13 due to https://bugs.eclipse.org/bugs/show_bug.cgi?id=485032
-// for some reason, the Gradle build does not seem to use the latest version of Xtext
-//			assertThat(size).isEqualTo(1)
+			assertThat(size).isEqualTo(1) // Integer.valueOf(1))
```
Comment (jjelschen) [inline org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend]: I guess the comment can be removed, right?

**Review by jjelschen — APPROVED** [review]

File: org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend (hunk 2 above)
Comment (gunther-bachmann) [reply org.testeditor.web.backend.xtext/src/test/java/org/testeditor/web/backend/xtext/index/IndexServiceClientIntegrationTest.xtend:49]: yep, still fails if removed

**Pull request merged**